### PR TITLE
update(no-deprecated-deno-api): deprecate `Deno.run`

### PR DESCRIPTION
**Pull request by petamoriken**
Blocked by https://github.com/denoland/deno/pull/17630

### CODE REVIEW

**Comment by petamoriken** [comment]
When https://github.com/denoland/deno/pull/17595 is landed, we should use it

File: docs/rules/no_deprecated_deno_api.md [hunk 1]
```diff
@@ -18,6 +18,13 @@ The IO APIs are already available in `std/io` or `std/streams`, so replace these
 deprecated ones with alternatives from `std`. For more detail, see
 [the tracking issue](https://github.com/denoland/deno/issues/9795).
 
+**Sub Process API**
+
+- `Deno.run`
+
+`Deno.run` was deprecated in faver of `Deno.Command`. See
```
Comment (magurotuna) [inline docs/rules/no_deprecated_deno_api.md:25]: typo
```suggestion
`Deno.run` was deprecated in favor of `Deno.Command`. See
```

Comment (petamoriken) [reply docs/rules/no_deprecated_deno_api.md]: thx!

**Review by magurotuna — APPROVED** [review]
LGTM - thank you so much!

**Pull request merged**